test/cases/ostree-simplified-installer.sh

```diff
@@ -885,6 +885,56 @@ EOF
 sudo ansible-playbook -v -i "${TEMPDIR}"/inventory -e image_type=redhat -e ostree_commit="${INSTALL_HASH}" -e edge_type=edge-simplified-installer -e fdo_credential="false" /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
 check_result
 
+########################################################################
+##
+## Build edge-simplified-installer with ignition embedded 
```

Comment:
This script is run on RHEL 8 as well but the feature was only added to RHEL 9.

Reply:
Thanks, will do